app/src/renderer/scripts/boot.js

```diff
@@ -51,7 +51,12 @@ export const startApp = async (
   enableGoogleAnalytics = _enableGoogleAnalytics
 ) => {
   Vue.use(Router)
-  Vue.use(Tooltip, { delay: 1 })
+  Vue.use(Tooltip, {
+    delay: 500,
+    placement: `bottom`,
```

Comment:
i think if you don't set this then it uses some kind of "auto" placement feature... so why always bottom?

Reply:
I will change it back to auto. There were some bugs when updating this library. I need to revert the update. Might be interesting to migrate to another more maintained lib.

Reply:
feel free to merge after changing this back to auto